Assignment 2.Rmd

```diff
@@ -86,9 +86,75 @@ plot(g,layout=layout.fruchterman.reingold, vertex.color=VERTEX$gender, edge.widt
 
 In Part II your task is to [look up](http://igraph.org/r/) in the igraph documentation and create a graph that sizes the student vertices in terms of the number of disciplinary actions they have recieved, and the teachers in terms of the number of disciplinary actions they have given out.
 
+```{r}
+
+NS<-group_by(EDGE,to)%>%summarise(time=sum(count))
+
+plot(g,layout=layout.fruchterman.reingold, vertex.color=VERTEX$gender, vertex.size=NS$time, edge.width=EDGE$count)
+
+```
```

Comment:
Very close. You have the right idea with the grouping and summing to get the counts. However, we want to do this for both teachers and students, and then merge the two data frames together, before creating the plot.